examples/LongClickExample.js

```diff
@@ -55,16 +55,14 @@ class LongClickExample extends React.Component {
     let columns = [];
 
     Object.keys(this.displayColumns).forEach((columnKey) => {
-      columns.push(
-        <Column
-          key={columnKey}
-          columnKey={columnKey}
-          flexGrow={2}
-          header={<DataCell>{columns[columnKey]}</DataCell>}
-          cell={(cell) => this.getCell(cell.rowIndex, cell.columnKey)}
-          width={100}
-        />
-      );
+      columns.push({
+        key: columnKey,
```

Comment:
I don't think `key` is required anymore. It was only specified as a special prop for React.
Could you remove usages here and elsewhere if applicable?